.gitignore

```diff
@@ -23,4 +23,7 @@ var
 .junitxml
 pyrightconfig.json
 .idea
-.python-version
\ No newline at end of file
+.python-version
+tests/data/collection-*
```

Comment:
This seems odd when are we excluding folders from tests

Reply:
My mistake.
I didn't intend to check this i